.jenkins/pipelines/standalone/measure-code-coverage.Jenkinsfile

```diff
@@ -1,6 +1,6 @@
 pipeline {
     agent {
-        label 'ACC-1804-DC4'
+        label 'Jenkins-Shared-DC2'
```

Comment:
I recommend not using shared nodes for anything that installs packages (i.e. the measure code coverage and report code coverage Jenkinsfiles).

Using shared nodes that will install global dependencies can cause a lot of issues down the road -- especially as we expand support for different versions, os, etc. The problems could be as simple as waiting (and failing) for apt lock, to a tangled web of broken dependencies.

Reply:
Yes, good catch. This was part of my tests, and I forgot to update it